Update cryptographic-functions-transact-sql.md - fix typo in function name

max-webster opened the pull request:
Fix typo 'VERIGYSIGNEDBYCERT' in function name. The link had the name spelled correctly so clicking on it still worked as expected.

max-webster committed:
Update cryptographic-functions-transact-sql.md

Fix typo 'VERIGYSIGNEDBYCERT' in function name. The link had the name spelled correctly so clicking on it still worked as expected.

opbld31 commented:
Learn Build status updates of commit _[f9660e1](https://github.com/max-webster/sql-docs/commits/f9660e17b5336f2411ee206bff3b696a0c97bbe2)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/t-sql/functions/cryptographic-functions-transact-sql.md](https://github.com/max-webster/sql-docs/blob/patch-2/docs/t-sql/functions/cryptographic-functions-transact-sql.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/42378d2c-d7bb-c607-bc95-176b26f971cc/202211300631123072-8328/BuildReport?accessString=614a38105318e0180fcabbe6cfdd59965e2ffa2bc272ae974520b4f4cbf996ef).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/42378d2c-d7bb-c607-bc95-176b26f971cc?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

max-webster commented:
I spotted another typo in the same file: VERIFYSIGNEDBYASMKEY should be VERIFYSIGNEDBYASYMKEY. Let me see if I can update or add another commit to the pull request here in the github web interface.

Commit:
Fix another typo in same file

Change VERIFYSIGNEDBYASMKEY to VERIFYSIGNEDBYASYMKEY.
Again, the link was OK, only the name of the function had the typo.

opbld31 commented:
Learn Build status updates of commit _[8b89818](https://github.com/max-webster/sql-docs/commits/8b89818bcad1d4d61f67da6f3f5ea0955a20d88b)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/t-sql/functions/cryptographic-functions-transact-sql.md](https://github.com/max-webster/sql-docs/blob/patch-2/docs/t-sql/functions/cryptographic-functions-transact-sql.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/42378d2c-d7bb-c607-bc95-176b26f971cc/202211300655085069-8328/BuildReport?accessString=f6bfcf27cfdf00caff3492f6dc0cfd2af221411ccbcc194f3fa0bbbf1a31ddcc).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/42378d2c-d7bb-c607-bc95-176b26f971cc?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

Court72 commented:
@VanMSFT 

Can you review the proposed changes? *IMPORTANT*: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

max-webster commented:
Just checking there's no holdup from me - is the change trivial enough that no CLA is required?

Court72 commented:
Hi @max-webster - No action for you at the moment. The CLA check has passed. Thanks!

Court72 commented:
@vanmsft 

Can you review this proposed update to your article and enter `#sign-off` in a comment if it's ready for merge?

VanMSFT commented:
Sorry for the delay.

#sign-off
